src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -36,8 +36,10 @@ const getStyles = ({ typography }) => ({
         gap: spacers.dp16,
     },
     title: {
-        ...typography.title,
-        paddingTop: spacersNum.dp16,
+        fontSize: '1.25rem',
+        color: colors.grey900,
+        fontWeight: 500,
+        paddingTop: spacersNum.dp8,
```

Comment:
I think, eventually, this should be replaced with either a `<PageTitle>` component, or an update to the `typography` definitions. This is the "base" fix that the standard fix should be based on.

Reply:
I agree! Does it make sense to create a follow-up Jira ticket for adding a `<PageTitle>` component on all the pages? 
Thanks!

Reply:
👍 here's the ticket: https://dhis2.atlassian.net/browse/DHIS2-14115